internal/rbd/controllerserver.go

```diff
@@ -120,7 +121,7 @@ func (cs *ControllerServer) parseVolCreateRequest(
 		return nil, status.Error(codes.InvalidArgument, "missing required parameter imageFeatures")
 	}
 
-	// if it's NOT SINGLE_NODE_WRITER and it's BLOCK we'll set the parameter to ignore the in-use checks
+	// if it's NOT SINGLE_NODE_SINGLE_WRITER and it's BLOCK we'll set the parameter to ignore the in-use checks
```

Comment:
why `SINGLE_NODE_WRITER` is removed from here?

Reply:
I wonder about this too. Is `SINGLE_NODE_WRITER` deprecated and replaced by `SINGLE_NODE_SINGLE_WRITER` maybe? If so, the commit message should explain that clearly.

Reply:
If it is depreciated, then we need to get rid of  https://github.com/ceph/ceph-csi/pull/2359/commits/cfe8f83c72724fe88f658943995dd97ea8098a11#diff-4bc3ac45c059d328076214abbf3b72db3cd475699833d9825fcaa6471a6ccd85R147 too.